lib/chewy/type/wrapper.rb

```diff
@@ -70,7 +70,7 @@ def highlight_name(method)
       end
 
       def attribute_defined?(attribute)
-        self.class.root_object && self.class.root_object.children.find { |a| a.name.to_s == attribute }.present?
+        self.class.root && self.class.root.children.find { |a| a.name.to_s == attribute }.present?
```

Comment:
The same here, when root object is not updated, there should be no need to run the code that is essentially a noop

Reply:
I want less exceptions tbh